composer.json

```diff
@@ -47,6 +47,7 @@
         "psr/log": "^1 || ^2 || ^3",
         "squizlabs/php_codesniffer": "3.7.1",
         "symfony/cache": "^4.4 || ^5.4 || ^6.0",
+        "symfony/uid": "^4.4 || ^5.4 || ^6.0",
```

Comment:
I'm not keen on adding that dependency, tbh. We should be able to reproduce the problem without it.

Reply:
That's fair. I only added it because it's technically supposed to be supported out of the box (but that might be a symfony concern, so it makes sense to not add it in doctrine)